lib/src/main/java/growthbook/sdk/java/GrowthBook.java

```diff
@@ -72,12 +83,44 @@ public GrowthBook() {
         this.conditionEvaluator = conditionEvaluator;
         this.experimentEvaluatorEvaluator = experimentEvaluator;
         this.context = context;
+
+        this.initializeEvalContext();
+    }
+
+    private void initializeEvalContext() {
+        // build options
+        Options options = Options.builder()
+                .enabled(this.context.getEnabled())
+                .isQaMode(this.context.getIsQaMode())
+                .allowUrlOverrides(this.context.getAllowUrlOverride())
+                .url(this.context.getUrl())
+                .stickyBucketIdentifierAttributes(this.context.getStickyBucketIdentifierAttributes())
+                .stickyBucketService(this.context.getStickyBucketService())
+                .trackingCallBackWithUser(new TrackingCallbackAdapter(this.context.getTrackingCallback()))
+                .featureUsageCallbackWithUser(new FeatureUsageCallbackAdapter(this.context.getFeatureUsageCallback()))
+                .build();
+
+        // build global
+        GlobalContext globalContext = GlobalContext.builder()
+                .features(this.context.getFeatures())
+                .savedGroups(this.context.getSavedGroups())
+                .build();
+
+        // build user context
+        UserContext userContext = UserContext.builder()
+                .attributes(this.context.getAttributes())
+                .stickyBucketAssignmentDocs(this.context.getStickyBucketAssignmentDocs())
+                .forcedVariationsMap(this.context.getForcedVariationsMap())
+                .build();
+
+        this.evaluationContext = new EvaluationContext(globalContext, userContext,
+                new EvaluationContext.StackContext(), options);
```

Comment:
This creates a new StackContext instance for every evaluation. we're not reusing it. It's just a static declaration in the EvaluationContext as it is an inner class.

Reply:
Aren't we re-using `this.evaluationContext` multiple times though?  If you evaluate 5 feature flags in a row, it will use the same context for all 5.  The stack context needs to be reset for every evaluation, otherwise it could cause conflicts.

Reply:
Ohh.. missed this file in the commit. just a sec.

Reply:
This is addressed.